python/src/vmaf/core/nn_train_test_model.py

```diff
@@ -5,7 +5,12 @@
 import numpy as np
 import scipy.stats
 from sklearn.metrics import f1_score
-import tensorflow as tf
+
+try:
+    import tensorflow as tf
```

Comment:
`tensorflow` is not declared as a requirement by vmaf, so this is either experimental code, or never used in tests
(however fails to even `import`, when doctests tries to discover stuff, it fails to even look at this file without this `try:` protection)